docs/Zähler.md

```diff
@@ -0,0 +1,80 @@
+
+# Zähler
+
+OpenWB benötigt zum erfolgreichen PV-Überschussladen die entsprechenden Zählerwerte am EVU-Punkt (EVU=Elektrizitätsversorgungsunternehmen), sprich dem Übergang ins öffentliche Netz. An dieser Stelle muss saldierend (in beide Richtungen) die Leistung pro Phase erfasst werden.
+
+Im einfachsten Fall geschieht dies durch Kauf und Einbau eines [EVU-Kits](##EVU-Kit). Sollten schon digital auslesbare Zähler vorhanden sein, so besteht die Möglichkeit diese Werte an OpenWB weiterzuleiten, auch mit Hilfe von Hausautomationsservern.
+
+Es gibt viele verschiedene Möglichkeiten Zähler, als auch Wechselrichter in das OpenWB-System einzufügen.  Die Struktur der Zähler muss dann im  [Lastmanagement](https://github.com/openWB/core/wiki/Lastmanagement-und-kaskadierte-Zähler) dem System bekanntgegeben werden. Hier können auch [virtuelle Zähler](##Virtuelle Zähler)  hinzugefügt werden, welche OpenWB-intern die darumherum angeordneten Zähler verrechnen.
+
+_Ich beschreibe hier zunächst den Weg, der für mich funktioniert bzw. die Erkenntnisse auf meinem Weg dahin. Weiteres bitte gern ergänzen._
+
+
+## EVU-Kit
+Das [EVU-Kit (Link zum Shop)](https://openwb.de/shop/?product=openwb-evu-kit) ist die einfachste Art in der Software einen Zähler an die OpenWB zu integrieren. Der Zähler muss von einem Elektriker direkt hinter dem Zähler des EVU in den Zählerschrank integriert werden. Es gibt, je nach Kapazität des Hausanschlusses, verschiedene Messvarianten des Zählers, welche sich von der Integration unterscheiden. Dies betrifft aber nur die Arbeit des Elektrikers.
+Der Zähler kommuniziert mit der OpenWB über Ethernet. Der Wallbox muss also nur noch in den Einstellungen die IP bekanntgegeben werden. Es gibt auch für Wechselrichter und Speicher entsprechende Kits.
+
+![EVU-Kit](pictures/EVU-PV-Speicher-Kit-689x1024.png)
+
+## MQTT
+
+OpenWB hat einen MQTT-Broker integriert, welcher unter Port 1883 erreichbar ist. Verschlüsselung oder Benutzerauthentifizierung ist deaktiviert und auch nicht aktivierbar. Ein Zähler, welche die benötigten Daten liefert muss sich mit diesem Broker verbinden und dort die Werte unter den entsprechenden Topics publishen.
```

Comment:
```suggestion
OpenWB hat einen MQTT-Broker integriert, welcher unter Port 1883 (ohne Verschlüsselung) und Port 8883 (mit Verschlüsselung) erreichbar ist.Benutzerauthentifizierung ist deaktiviert und auch nicht aktivierbar. Ein Zähler, welche die benötigten Daten liefert muss sich mit diesem Broker verbinden und dort die Werte unter den entsprechenden Topics publishen.
```

Reply:
Kleiner Typo drin. Ist in der neuen Version korrigiert